scala/scala_import.bzl

```diff
@@ -13,9 +13,10 @@ def _scala_import_impl(ctx):
         intellij_metadata,
     ) = (target_data.code_jars, target_data.intellij_metadata)
     current_jars = depset(current_target_compile_jars)
-    exports = _collect(ctx.attr.exports)
-    transitive_runtime_jars = _collect_runtime(ctx.attr.runtime_deps)
-    jars = _collect(ctx.attr.deps)
```

Comment:
I wanted to be sure to call this line out: as it stands in master, this `jars` is unused and is deleted here.